### ISSUE TITLE
[Bug] Safari on IOS missing topbar

### ISSUE THREAD

**Issue by xInsertx** [jfurrow/flood#348]
Running iOS 10.3

Don't see the top bar. Sometimes I see part of it.

![img_0268](https://cloud.githubusercontent.com/assets/10349764/25801446/f7b4ec6e-3430-11e7-81f4-5339885e2c17.PNG)
![img_0269](https://cloud.githubusercontent.com/assets/10349764/25801445/f7b2cd9e-3430-11e7-87e4-844ec7ee76a2.PNG)

**Comment by jfurrow** [comment]
Yeah, admittedly I have only been testing Flood in Chrome. This will be trivial to fix, just need to get around to it...

**Comment by geo5275** [comment]
This also have been happening for a while on Chrome for Android .

I'm running Chrome 58.0.3029.83 and Android 7.1.2

![screenshot_20170513-214012](https://cloud.githubusercontent.com/assets/20895147/26030295/c6727290-3825-11e7-8ed7-2fddc81b6505.png)
![screenshot_20170513-214040](https://cloud.githubusercontent.com/assets/20895147/26030294/c6726f20-3825-11e7-88e5-c858b9d7ae67.png)

**Comment by disrupted** [comment]
has this been resolved? currently running into a similar issue with mobile Safari on iOS 12.1
